vision_datasets/common/balanced_instance_weights_generator.py

```diff
@@ -0,0 +1,63 @@
+"""Instance weights generator for DataManifest. Only works for classification, detection, multitask. Adapted from irispy:
```

Comment:
This is an open-sourced repo supposed to be accessible by any one

Reply:
sth like "Generate instance weights, which can be used for balancing the dataset, by sampling instances based on the weights, etc"